CI_JobHistory.py

```diff
@@ -168,6 +168,9 @@ def display_ci_links(config_data):
             j=j+1
             ci_name_list.append(ci_name)
             print(j,'',ci_name)
+        j=j+1
+        print(j, " All")
```

Comment:
Should we change this to "All of the above"?

Reply:
Sure We can do it.